code/game/objects/items/cards_ids.dm

```diff
@@ -1158,6 +1164,9 @@
 
 /obj/item/card/id/advanced/prisoner/examine(mob/user)
 	. = ..()
+	if(!user.can_read(src))
```

Comment:
```suggestion
	if(!.)
```

Reply:
Stuff like this is really ugly to look at codewise compared to other langauges.